xpk.py

```diff
@@ -66,7 +66,8 @@
 default_docker_image = 'python:3.10'
 default_script_dir = os.getcwd()
 default_gke_version="1.29.1-gke.1589017"
-xpk_current_version = "0.3.0"
+__version__ = "0.3.0"
```

Comment:
nit to add a comment saying __version__ shares the xpk version with pyproject / pip installation.